pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java

```diff
@@ -180,6 +180,10 @@
                 "used for handling connections to brokers, default is 1 thread")
         public int ioThreads = 1;
 
+        @Parameter(names = {"-listenerThreads", "--num-listener-threads"}, description = "Set the number of threads"
+                + " to be used for message listeners")
+        public int listenerThreads = 1;
```

Comment:
The short option `-xx` usually doesn't need to contain the full name. See example

```
        @Parameter(names = { "-sp", "--subscription-position" }, description = "Subscription position")
        private SubscriptionInitialPosition subscriptionInitialPosition = SubscriptionInitialPosition.Latest;
```

The short option is `-sp` and the long option is `--subscription-position`.

Reply:
Ok, thanks for your advice, is it all right now?